scripts/cc-cli/src/test/unit-tests/collect-coins.test.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  isExternalAddressValid,
+  isTxHashValid,
+} from "../../commands/collectCoins";
+
+describe("Test isTxHashValid", () => {
+  test("success", () => {
+    expect(
+      isTxHashValid(
+        "0x2446f1fd773fbb9f080e674b60c6a033c7ed7427b8b9413cf28a2a4a6da9b56c",
+      ),
+    ).toBeTruthy();
+  });
+
+  test("Fails with missing prefix", () => {
+    expect(
+      isTxHashValid(
+        "2446f1fd773fbb9f080e674b60c6a033c7ed7427b8b9413cf28a2a4a6da9b56c",
+      ),
+    ).toBeFalsy();
+  });
+
```

Comment:
Missing scenario "should return false when argument contains non-hex characters"

and the actual test data contains 64 chars, with a `0x` prefix, however some of them are not hex characters (e.g. `INVALID-value`).